tests/02-compile-arm-ports/Makefile

```diff
@@ -107,8 +107,6 @@ platform-specific/cc26x0-cc13x0/cc26x0-web-demo/cc26x0-cc13x0:BOARD=sensortag/cc
 platform-specific/cc26x0-cc13x0/base-demo/cc26x0-cc13x0:BOARD=sensortag/cc2650 \
 platform-specific/cc26x0-cc13x0/base-demo/simplelink:BOARD=sensortag/cc1350 \
 platform-specific/cc26x0-cc13x0/base-demo/simplelink:BOARD=sensortag/cc1352r1 \
-platform-specific/nrf/start-network-core/nrf:BOARD=nrf52840/dk \
-platform-specific/nrf/start-network-core/nrf:BOARD=nrf52840/dongle \
 platform-specific/nrf/start-network-core/nrf:BOARD=nrf5340/dk/application \
 platform-specific/nrf/start-network-core/nrf:BOARD=nrf5340/dk/network \
```

Comment:
Line 113 should also be removed. The network core can not start itself.

Reply:
Maybe this is the comment part you don't understand, removing that line removes all the testing of `TARGET_ONLY`/`BOARD_ONLY` functionality.

Reply:
Yes, I did not understand this was the intended test. A comment about this would be nice but probably not easy to add close to the relevant lines.

Reply:
Yes, a comment will end up ~100 lines earlier.